website/frontend/templates/docs/plugin-api.html

```diff
@@ -66,7 +76,10 @@ <h1 id="tagger_script_functions">Tagger Script Functions</h1>
       <p>
         To define new
         <a href="/docs/scripting/" title="Picard Qt/Scripting">tagger script</a> function use
```

Comment:
s/function/functions/, if you don't mind fixing that too. (or s/new/a new/ above.)